### DISCUSSION TITLE
Remove login

### DISCUSSION THREAD

**Discussion by noahfraiture** [open-webui/open-webui#5753] [Q&A] [answered]
Hello,
Is it possible to remove the login system ? I don't want to add to put a password, i'm the only person that can access the service

**Comment by tjbck** [comment, marked as answer]
https://docs.openwebui.com/

**Reply by noahfraiture** [reply]
thank you, I didn't see

**Reply by FatheredPuma81** [reply]
I'd say this is why AI is a thing but apparently none of them can scrap this crappy Wiki that I'm not reading for something that deserves a literal button.
"You can't turn off authentication because there are existing users. If you want to disable WEBUI_AUTH, make sure your web interface doesn't have any existing users and is a fresh installation." LMAO adding users without permission is some backdoor level crap right there.

**Reply by Solenhal** [reply]
It doesn't showcase how to do this with the **Python** install method, only the **Docker** install method.

`docker run -d -p 3000:8080 -e WEBUI_AUTH=False -v open-webui:/app/backend/data --name open-webui ghcr.io/open-webui/open-webui:main` as shown in the quick-start doesn't work for my use case.
